src/Components/Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.npmproj

```diff
@@ -22,8 +22,8 @@
           Outputs="@(WebPackOutputs)">
     <RemoveDir Directories="dist" />
 
-    <Yarn Command="run build:debug" StandardOutputImportance="High" StandardErrorImportance="High" />
-    <Yarn Command="run build:production" StandardOutputImportance="High" StandardErrorImportance="High" />
+    <Yarn Command="run build:debug" />
+    <Yarn Command="run build:production" />
```

Comment:
Won't these now succeed even if the command writes to `stderr`, (likely) indicating a problem? That is, @javiercn didn't you add the `*Importance` attributes for reasons?

Reply:
This doesn't change how stderr is handled, it's removing code that's unnecessary with the new version of Yarn. Per feedback from @BrennanConroy and @javiercn , I updated the Yarn task in 1.15.2 to set the default logging level to 'High'.  See https://github.com/natemcmaster/Yarn.MSBuild/commit/0c628a75c91b0d10f49102ff7289f10792dbe20a

'High' is the preferred default now because yarn output will appear in console logs, and binlogs are huge and a pain to work with.